### ISSUE TITLE
Typos in examples of fractional-milliseconds

### ISSUE THREAD

**Issue by ao2** [w3c/ttml-webvtt-mapping#29]
I think there are some typos in the examples about fractional-milliseconds.

Here the text should probably refer to `milliseconds.fractional-milliseconds` instead of `milliseconds.fractional-seconds`:
https://github.com/w3c/ttml-webvtt-mapping/blob/88808310e66c15feb67d7003bca532f1cc72a262/index.html#L3192

And the description seems to be copied from the minutes conversion:
https://github.com/w3c/ttml-webvtt-mapping/blob/88808310e66c15feb67d7003bca532f1cc72a262/index.html#L3198

Which is probably copied from the hours conversion and may also need some fixing:
https://github.com/w3c/ttml-webvtt-mapping/blob/88808310e66c15feb67d7003bca532f1cc72a262/index.html#L3094

And here the fractional part seems to be missing:
https://github.com/w3c/ttml-webvtt-mapping/blob/88808310e66c15feb67d7003bca532f1cc72a262/index.html#L3223

And the calculation seems to be the result of `ceil(3.45) / 1000` ? Is this how the rounding is supposed to work?

**Comment by nigelmegitt** [comment]
Thanks for these, and apologies for the slow response. I'm not sure when TTWG will be able to address these issues; just for information this document is based on a version of WebVTT from some years ago and probably needs a wider review to bring it into line with the current WebVTT specification.

**Comment by ao2** [comment]
I see, feel free to close the issue if appropriate.

**Comment by nigelmegitt** [comment]
Not at all, I think the issue looks appropriate and should be fixed! I'm just not sure _when_ it will be fixed, sadly.

**Comment by gkatsev** [comment]
I agree that this should be fixed but as Nigel said, the first step here would be to revamp the spec against the current version of WebVTT.
